core/model/pom.xml

```diff
@@ -8,13 +8,25 @@
 	</parent>
 	<artifactId>rdf4j-model</artifactId>
 	<name>RDF4J: Model</name>
-	<description>RDF model interfaces and implementations.</description>
+	<description>RDF model implementations.</description>
 	<dependencies>
+		<dependency>
+			<groupId>${project.groupId}</groupId>
+			<artifactId>rdf4j-model-api</artifactId>
+			<version>${project.version}</version>
+		</dependency>
 		<dependency>
 			<groupId>${project.groupId}</groupId>
 			<artifactId>rdf4j-util</artifactId>
 			<version>${project.version}</version>
 		</dependency>
+		<dependency>
+			<groupId>${project.groupId}</groupId>
+			<artifactId>rdf4j-model-api</artifactId>
+			<version>${project.version}</version>
+			<type>test-jar</type>
```

Comment:
This won't work unless the model-api module actually produces and publishes a test jar. I'm not sure yet if that is desirable - we use this approach in one other place (the rio api) but for everything else we have set up separate test suite projects instead. I'll take a look at this and if necessary refactor.